setup.py

```diff
@@ -55,7 +55,6 @@ def package_files(directory):
         "sqlparse>=0.3.1",
         # Required to run the MLflow server against SQL-backed storage
         "sqlalchemy",
-        "gorilla",
```

Comment:
Accidentally re-included this while resolving merge conflicts in https://github.com/mlflow/mlflow/pull/3582. It should be dropped.